### PR TITLE
Fixed doctstring porkchop

### PR DESCRIPTION
**Pull request by priyanshurohilla**
Addresses #892

### CODE REVIEW

**Comment by jorgepiloto** [comment]
Thank you for addressing this @priyanshurohilla. It seems that your docstrings contain both whitespaces and tabs. I checkout out it locally and running `tox -e reformat` will not fix this, so you must correct it manually.

**Comment by priyanshurohilla** [comment]
ok i am doing it

**Comment by priyanshurohilla** [comment]
> Thank you for addressing this @priyanshurohilla. It seems that your docstrings contain both whitespaces and tabs. I checkout out it locally and running `tox -e reformat` will not fix this, so you must correct it manually.

what should i do to correct it manually?

File: src/poliastro/plotting/porkchop.py [hunk 1]
```diff
@@ -138,6 +138,10 @@ def porkchop(
 
     Returns
     -------
+    dv_launch: np.ndarray
+    Launch delta V 
+    dv_arrival: np.ndarray
+    Arrival delta v
```
Comment (iamabhishek0) [inline src/poliastro/plotting/porkchop.py:144]: See line 145 and 146 and try to indentate like that.

**Comment by jorgepiloto** [comment]
While your changes will pass the tests now the git history became weird:

```bash
git log --graph --decorate --all --oneline
```
Will prompt:

```bash
*   f555544 (HEAD -> docstring, priyanshurohilla/docstring) Fixed doctstring porkchop
|\  
| * 168cba5 Updated porkchop
* | 2be49d7 Updated docstring fix
|/  
* 5ec6ab0 Fixed doctstring porkchop
```

There should be only one line of commits. You can either squash or run:

```bash
git reset --soft 5ec6ab0
git add . && git commit --amend && git push -f
```

File: src/poliastro/plotting/porkchop.py (hunk 1 above)
Comment (priyanshurohilla) [reply src/poliastro/plotting/porkchop.py]: Thanks for the suggestion, but i figured out myself. Actually there was some other problem not this. My editor was making some problems and i had to change some settings on it and then it worked out :)

**Comment by priyanshurohilla** [comment]
> While your changes will pass the tests now the git history became weird:
> 
> ```shell
> git log --graph --decorate --all --oneline
> ```
> 
> Will prompt:
> 
> ```shell
> *   f555544 (HEAD -> docstring, priyanshurohilla/docstring) Fixed doctstring porkchop
> |\  
> | * 168cba5 Updated porkchop
> * | 2be49d7 Updated docstring fix
> |/  
> * 5ec6ab0 Fixed doctstring porkchop
> ```
> 
> There should be only one line of commits. You can either squash or run:
> 
> ```shell
> git reset --soft 5ec6ab0
> git add . && git commit --amend && git push -f
> ```After doing this , one another screen open. There i should `write out` right?

**Comment by jorgepiloto** [comment]
Sorry for the making a `git push --force` to your branch but this should solve the git branching issues we encountered :sweat_smile:

**Comment by jorgepiloto** [comment]
I noticed that docstring return parameters were not placed in the correct order, pushing again and hope to close this one the CI approves.

**Comment by priyanshurohilla** [comment]
> I noticed that docstring return parameters were not placed in the correct order, pushing again and hope to close this one the CI approves.

Ohh i didn't know that they have to be in the right order! I'll keep that i mind from now onwards😊

**Comment by jorgepiloto** [comment]
> Something weird has happened here folks, the commit appears as attributed to @jorgepiloto, but I understand that it's not the case, right?

The git branch was a mess and I forced-push the commit, sorry :sweat_smile:

**Comment by astrojuanlu** [comment]
Anyway, it was a two line change, so let's merge it :)

@priyanshurohilla next time if you have problems with git, please go to #poliastro-newbies so everybody learns in the process.

**Pull request merged**